packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -393,6 +393,8 @@ class AppDomain extends Domain {
 
     if (options.debuggingEnabled) {
       connectionInfoCompleter = new Completer<DebugConnectionInfo>();
+      // Callback hookups' Futures are uninteresting.
+      // ignore: unawaited_futures
       connectionInfoCompleter.future.then<Null>((DebugConnectionInfo info) {
```

Comment:
@danrubel @devoncarew I'm actually not sure what the right behaviour is either. 

What should we do if the callback here fails? Should we send a 'stop' with 'error' json like below? Should we re-throw?